common/cmake_modules/GncAddTest.cmake

```diff
@@ -109,7 +109,20 @@ endfunction()
 
 function(gnc_add_scheme_test _TARGET _SOURCE_FILE)
   add_test(${_TARGET} ${CMAKE_COMMAND} -E env
-    ${GUILE_EXECUTABLE} --debug -c "(load-from-path \"${_TARGET}\")(exit (run-test))"
+    ${GUILE_EXECUTABLE} --debug -c "
+    (format #t \"%load-path =~%\")
+    (for-each (lambda(x)(format #t \"~s~%\" x)) %load-path)
+    (format #t \"%load-compiled-path =~%\")
+    (for-each (lambda(x)(format #t \"~s~%\" x)) %load-compiled-path)
```

Comment:
This will add a lot of noise to test output. How about making it conditional on loading a file from outside build-dir?